src/components/NFTBook/CoverWithFrame.vue

```diff
@@ -4,15 +4,15 @@
     :class="[
       'group',
       'relative',
-      'aspect-[4/5]',
+      'aspect-[1]',
```

Comment:
Since the new home page & listing page are using `aspect-4/5`, I suggest using an extra prop or `tailwind-merge` lib for better control

Reply:
[💄 Add props for CoverWithFrame](https://github.com/likecoin/liker-land/pull/1588/commits/8a74d7a4768494c3bf21ac728d727843408b2f0e)